tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -246,14 +245,15 @@ Bridge network - ESX none
     Cleanup VIC Appliance On Test Server
 
 Bridge network - invalid
-    Pass execution  asdf
+    Pass execution  Skipping test
```

Comment:
The test assumes the port group 'AAAAA' won't be created, but it will on ESX hosts. Do you want me to fix the test so it checks for a successful create?